ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_network_services.schema.yml

```diff
@@ -938,6 +938,12 @@ $defs:
                 type: bool
                 default: true
                 description: Extend this L2VLAN over VXLAN.
+              evpn_vlan_bundle:
+                type: str
+                description: |
+                  Define a vlan-aware-bundle name which should be used instead of the vlan name. If several vlans have the same evpn_vlan_bundle defined they will be grouped together.
+                  The RD and RT will be used from the lowest VLAN id defined in the same evpn_vlan_bundle.
+                  To use this option the global evpn_vlan_aware_bundles option must be set to true.
```

Comment:
We should probably catch this in the code, and add a negative unit test for this.

Reply:
Not sure I get what you mean, what do you want to catch in the code?

Reply:
If someone sets the evpn bundle name but the global evpn_vlan_aware_bundles is not set, we should error out in the code.

Reply:
I added a check in the code and also a negative test, can you have a look?

Reply:
thank you that looks good!